### PR TITLE
dont allow multiple popup pushes

### PR DESCRIPTION
**Pull request by chrisnojima**
Normally we get the route state from the routeTree own props but the connected messages don't do this. We could pass that down (and maybe thats cleaner) but the way it was was that it derived it itself from the store. Problem is, once you actually push the popup, the connected gets called again and its own idea of its routestate gets updated incorrectly so the message thinks its in /chattabs/selectedconversation/attachment. This makes the navigateup (which uses the new putActionIfOnPath) fail cause it thinks it hasn't naved.

tldr; when the message derives it's routeState, it only extracts /chattab/selectedconversation. Goodnews is that this is the only place in the code where we do this

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump this is a 1 liner
